utils.py

```diff
@@ -471,3 +471,19 @@ def get_asset_dir_prefix():
             ASSET_DIR_PREFIX = '/build/%s' % cache_slug
 
     return ASSET_DIR_PREFIX
+
+TEMPLATE_DIR_PREFIX = None
+def get_template_dir_prefix():
+    """Returns prefix for template directory depending whether dev or prod.
+    It is used as a prefix in urls for script files.
+    """
+    global TEMPLATE_DIR_PREFIX # pylint: disable=global-statement
+    if not TEMPLATE_DIR_PREFIX:
+        TEMPLATE_DIR_PREFIX = '/templates/dev/head'
```

Comment:
Also I suggest that you prefix the global constant _CACHE_SLUG with an underscore to indicate that it's private to this module and should not be directly referenced from elsewhere.